### ISSUE TITLE
get Direction and Distance

### ISSUE THREAD

**Issue by fohnbit** [fohnbit/radarCat#3]
Hello @erikmansson,

is it possible to easy recognize the direction of the movement, in your example code?
examples/processing/sparse_speed.py

I just want to know, is the object moving approaches or removes to the sensor.

And maybe I get the current distance of the current max speed?

Thank you!

**Comment by erikmansson** [comment]
From _one_ frame of sparse data, it is unfortunately not possible to obtain the direction of the speed. This has to do with the sparse data being purely real (not complex).

Yes, you can get the distance. Now, it's discarded by
https://github.com/fohnbit/radarCat/blob/6e1b03ae520d75929898b40813e851f783bb8609/examples/processing/sparse_fft.py#L115

To get it, you could instead do something like
`max_depth_index, max_bin = np.unravel_index(abs_fft.argmax(), abs_fft.shape)`

See https://github.com/numpy/numpy/issues/9283#issuecomment-310095393 if you're unsure of what that snippet does.

To convert the depth index to depth in meters you can use
https://github.com/fohnbit/radarCat/blob/be9251f767e3a6b19f0ee52406c408f4925add9c/examples/processing/sparse_fft.py#L188

Basically just
```
self.depths = get_range_depths(sensor_config, session_info)  # do this in the processor __init__

depth = self.depths[max_depth_index]  # and this in the process
```

Ok so now you have a crude distance estimate. This could be used to figure out if the object is moving towards or away from the sensor. We're working on how to do this in a robust way, but a simple way would be to check if this distance is increasing or decreasing over frames.

**Comment by fohnbit** [comment]
Hello ... works fine, thank you!
